web/libs/editor/src/components/Comments/Comments.tsx

```diff
@@ -1,14 +1,17 @@
 import { type FC, useEffect } from "react";
 import { observer } from "mobx-react";
 import { Block } from "../../utils/bem";
-import { CommentsList } from "./CommentsList";
 import { CommentForm as CommentFormOld } from "./OldComment/CommentForm";
 import { CommentForm as CommentsFormNew } from "./Comment/CommentForm";
+import { CommentsList as CommentsListOld } from "./OldComment/CommentsList";
+import { CommentsList as CommentsListNew } from "./Comment/CommentsList";
 import { useMounted } from "../../common/Utils/useMounted";
 import { FF_DEV_3034, FF_PER_FIELD_COMMENTS, isFF } from "../../utils/feature-flags";
 
 import "./Comments.scss";
+
 const CommentForm = isFF(FF_PER_FIELD_COMMENTS) ? CommentsFormNew : CommentFormOld;
+const CommentsList = isFF(FF_PER_FIELD_COMMENTS) ? CommentsListNew : CommentsListOld;
```

Comment:
minor nitpick - since `isFF(FF_PER_FIELD_COMMENTS)` is reused - lets check it once and reuse the calculated value